ios/Classes/SwiftQonversionFlutterSdkPlugin.swift

```diff
@@ -123,6 +126,21 @@ public class SwiftQonversionFlutterSdkPlugin: NSObject, FlutterPlugin {
     }
   }
   
+  private func offerings(_ result: @escaping FlutterResult) {
+    Qonversion.offerings { offerings, error in
+      if let error = error {
+        result(FlutterError.offeringsError(error.localizedDescription))
+      }
+      
+      guard let offerings = offerings else {
+        return result(nil)
+      }
+      
+      
+      result(offerings.toMap().toJson())
```

Comment:
using json encoding when passing data to Dart in order to avoid Flutter problems when parsing complex Maps coming from native side. 
Will explain details if needed